### PR TITLE
Replace icon font with SVG icons (#45)

### PR DESCRIPTION
**Pull request by robinst**
This shaves off about 230 KB of page weight and is one less domain to
load resources from.

The .svg files except email.svg are from the Simple icons project:
https://simpleicons.org/

email.svg is from Octicons:
https://octicons.github.com/

### CODE REVIEW

File: dev/less/site.less [hunk 1]
```diff
@@ -220,8 +213,8 @@ section.icons {
         a {
             margin-left: 6px;
             margin-right: 6px;
-            color: @compensated-black;
-            .transition(~'color @{transition-speed-1} @{in-sine}');
+            opacity: 0.6;
```
Comment (robinst) [inline dev/less/site.less:216]: The icons use black as color, so we can use the opacity instead.

File: dev/less/site.less [hunk 2]
```diff
@@ -258,19 +248,6 @@ section.icons {
             a {
                 margin-left: 3px;
                 margin-right: 3px;
-                color: @gray;
-
-                &:hover {
-                    color: @black;
```
Comment (robinst) [inline dev/less/site.less:264]: Not sure about these colors.

File: static/img/email.svg [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="14px" height="16px" viewBox="0 0 14 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 40.3 (33839) - http://www.bohemiancoding.com/sketch -->
+    <title>mail</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
```
Comment (robinst) [inline static/img/email.svg:6]: Just saw these now, maybe we should optimize this one.

File: theme.toml [hunk 4]
```diff
@@ -3,7 +3,7 @@ license     = "MIT"
 licenselink = "https://github.com/nishanths/cocoa-hugo-theme/blob/master/LICENSE"
 description = "Consistent, responsive theme with clean typography"
 homepage    = "https://github.com/nishanths/cocoa-hugo-theme"
-tags        = ["open-sans", "minimal", "clean", "ionicons", "comments"]
+tags        = ["open-sans", "minimal", "clean", "svg", "comments"]
```
Comment (robinst) [inline theme.toml:6]: Is that a tag? :)

File: static/img/email.svg (hunk 3 above)
Comment (nishanths) [reply static/img/email.svg:6]: OK for me to merge as it is now. This is already a huge improvement compared to the current setup. :)

File: theme.toml (hunk 4 above)
Comment (nishanths) [reply theme.toml:6]: LOL

**Comment by nishanths** [comment]
@robinst Thank you the changes, and for annotating with comments.
We can revisit the `email` svg later if needed.

All of the changes look good to me. Merged.

> This shaves off about 230 KB of page weight and is one less domain to
load resources from.

So much yes!

**Pull request merged**